kubert/src/admin.rs

```diff
@@ -238,3 +250,22 @@ fn handle_ready(Readiness(ready): &Readiness, req: Request<Body>) -> Response<Bo
             .unwrap(),
     }
 }
+
+fn handle_metrics(
+    metrics: &PrometheusHandle,
+    process: &Collector,
+    req: Request<Body>,
+) -> Response<Body> {
+    process.collect();
+    match *req.method() {
+        hyper::Method::GET | hyper::Method::HEAD => Response::builder()
+            .status(hyper::StatusCode::OK)
+            .header(hyper::header::CONTENT_TYPE, "text/plain")
+            .body(metrics.render().into())
+            .unwrap(),
+        _ => Response::builder()
+            .status(hyper::StatusCode::METHOD_NOT_ALLOWED)
```

Comment:
[according to RFC 9110](https://httpwg.org/specs/rfc9110.html#status.405), when sending a 405 Method Not Allowed status,
> The origin server MUST generate an [Allow](https://httpwg.org/specs/rfc9110.html#field.allow) header field in a 405 response containing a list of the target resource's currently supported methods

so we should probably add:
```suggestion
            .status(hyper::StatusCode::METHOD_NOT_ALLOWED)
            .header(hyper::header::ALLOW, "GET, HEAD")
```
in order to be actually standards compliant